src/api/Ledger.ts

```diff
@@ -25,7 +26,15 @@ export const findCurrencyExplorer = (
   }
 
   if (config.experimental && getEnv("EXPERIMENTAL_EXPLORERS")) {
-    const { base, version } = config.experimental;
+    const base = config.experimental.base;
+    let version = config.experimental.version;
+    //V2 explorer for doge and bitcoin cash when libcore is used
+    if (
+      (currency.id === "bitcoin_cash" || currency.id === "doge") &&
+      !shouldUseJS(currency)
+    ) {
+      version = "v2";
```

Comment:
shouldn't bitcoin_cash + V2 also to set the explorer id to abc ?

Reply:
1. Confirmed by backend team, "abc" and "bch" are exactly same for v2/v3. it doesn’t matter whether we use abc/bch.
2. In fact, for the old implementation, the id doesn’t impact the url. (but the version does)
for the old implementation, it seems that the “id” used by explorer url (abc) depends on the libcore
https://github.com/LedgerHQ/lib-ledger-core/blob/fc9d762b83fc2b269d072b662065747a64ab2816/core/src/wallet/bitcoin/networks.cpp#L93
So, it doesn't break anything I think